javascript/javascript3/week1/weather.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;0,700;0,800;1,300;1,600;1,700;1,800&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <title>Homework js3 week1</title>
+    
+</head>
+<body>
+    <h1 id="weather-title">Weather app</h1>
+    <section class="weather-wrapper">
+        <form class="search-weather">
+<input type="text" id="city-input" placeholder="city name" onclick="this.value=' '" >
+<button id="btn" type="submit"><i class="fa fa-search"></i></button>  
+</form>
+<p id="error-p"></p>
+<div class="weather loading">
+<h2 class="city">Weather in Roskilde</h2>
```

Comment:
It doesn't show your local weather on your page. If you would like to make it, could you please add a button on your Js file as it's mentioned in the homework, you can find the location on the [geo location api](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API) link that has mentioned on your homework.

https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API/Using_the_Geolocation_API#examples

Reply:
that was the optional part I tried to do it but Im not sure how to connect things together